### PR TITLE
Change pack incident fields of type multiSelect with no select values to array

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/44751

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/Asset/ReleaseNotes/1_0_3.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#### Maintenance and stability enhancements for the following fields:
+
+#### Incident Fields
```
Comment (reutshal) [inline Packs/Asset/ReleaseNotes/1_0_3.md:3]:
```suggestion
#### Incident Fields
Maintenance and stability enhancements for the following fields:
```

Comment (reutshal) [reply Packs/Asset/ReleaseNotes/1_0_3.md:3]: Please change all RN accordingly

File: Packs/IntegrationsAndIncidentsHealthCheck/IncidentFields/incidentfield-PlaybooksFailedCommands.json [hunk 2]
```diff
@@ -17,9 +17,7 @@
  "neverSetAsRequired": false,
  "ownerOnly": false,
  "required": false,
- "selectValues": [
-  ","
- ],
```
Comment (reutshal) [inline Packs/IntegrationsAndIncidentsHealthCheck/IncidentFields/incidentfield-PlaybooksFailedCommands.json:22]: I see there was a value here, please consult with @altmannyarden before making this change and make sure it doesn't effect the use case

**Review by reutshal — CHANGES_REQUESTED** [review]

File: Packs/Asset/ReleaseNotes/1_0_3.md (hunk 1 above)
Comment (dansterenson) [reply Packs/Asset/ReleaseNotes/1_0_3.md]: Done

**Review by reutshal — APPROVED** [review]

File: Packs/IntegrationsAndIncidentsHealthCheck/IncidentFields/incidentfield-PlaybooksFailedCommands.json (hunk 2 above)
Comment (dansterenson) [reply Packs/IntegrationsAndIncidentsHealthCheck/IncidentFields/incidentfield-PlaybooksFailedCommands.json:22]: Checked, works fine

**Pull request merged**